Source/SwiftLintBuiltInRules/Rules/Lint/AsyncWithoutAwaitRule.swift

```diff
@@ -1,6 +1,8 @@
 import SwiftLintCore
 import SwiftSyntax
 
+// swiftlint:disable file_length
```

Comment:
We usually move the examples into a separate file if the actual rule becomes too long.